docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -91,48 +77,47 @@ Make sure to also review our [app development best practices](/docs/integrations
-* During the OAuth flow the app requested the merchant to authorize read and write permissions on the Information and Settings API scopes (see [Installing the app](#installing-the-app)).
-* During the OAuth flow the `client_id` and `access_token` was received and stored by the tax provider (see [Installing the app](#installing-the-app)).
+You will be ready to establish a connection with the Tax Provider API when all of the following are true:
 
-For context, the [Tax Provider API Connection endpoints](/docs/rest-contracts/tax-app-connection) provide an added layer of security for tax providers. They set basic authentication credentials for the tax provider and these basic credentials are used to authenticate each API request to the tax provider from the associated store.
+* You have shared your tax provider details with BigCommerce through email, and we've replied confirming the tax provider configuration is now ready on all supplied merchant and/or partner test stores (see [Sharing provider details with BigCommerce](#sharing-provider-details-with-bigcommerce)).
+* BigCommerce has shared the `provider_id` of the tax provider configuration with you through email (see [Sharing provider details with BigCommerce](#sharing-provider-details-with-bigcommerce)).
+* You have built a draft app that successfully handles the OAuth flow during app installation (see [Building the app](#building-the-app)).
+* During the OAuth flow the app requests that the merchant authorize read and write permissions on the Information and Settings API scopes (see [Installing the app](#installing-the-app)).
+* During the OAuth flow the `client_id` and `access_token` was received and stored by you (see [Installing the app](#installing-the-app)).
 
-If the tax provider supports all eligible stores, then they may choose to provide an account registration flow in their app iFrame in order to capture these basic authentication credentials from merchants. Learn more about designing the app UI [here](/docs/integrations/apps/guide/ui).
+For context, the [Tax Provider API Connection endpoints](/docs/rest-contracts/tax-app-connection) provide an added layer of security for tax provider connectivity. These basic credentials are used to authenticate each API request to the tax provider from the associated store, allowing you to reject any other unauthorized requests.
 
-If the tax provider is a private instance, then they may choose to provide the basic authentication credentials themselves.
+An account login flow allows you to capture basic authentication credentials from merchants. It's best to provide a login flow within your app iFrame so that you can choose your own authentication method. You can also provide an account registration flow in your iFrame to help merchants create a new tax provider account. Learn more about [designing the app UI](/docs/integrations/apps/guide/ui).
```

Comment:
```suggestion
An account login flow lets you capture basic authentication credentials from merchants. We recommend that you provide a login flow within your app iFrame so that you can choose your own authentication method. You can also provide an account registration flow in your iFrame to help merchants create a new tax provider account. Learn more about [designing the app UI](/docs/integrations/apps/guide/ui).
```